### PR TITLE
Update CI files and set python_version to 3.6

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: .github/workflows/scripts/before_install.sh [hunk 1]
```diff
@@ -53,6 +53,10 @@ echo "legacy_component_name: pulpcore" >> .ci/ansible/vars/main.yaml
 echo "component_name: core" >> .ci/ansible/vars/main.yaml
 echo "component_version: '${COMPONENT_VERSION}'" >> .ci/ansible/vars/main.yaml
 
+
+echo "ci_base: pulp/pulp-ci-centos:python36" >> .ci/ansible/vars/main.yaml
```
Comment (mdellweg) [inline .github/workflows/scripts/before_install.sh:57]: `pulp_container_tag: python36` should do the trick.
`ci_base` was originally meant to be set from the commit message (like Required-PR).

Comment (daviddavis) [reply .github/workflows/scripts/before_install.sh]: Done. Can you look at https://github.com/pulp/plugin_template/pull/459 ?

File: template_config.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 # were not present before running plugin-template have been added with their default values.
 
+# generated with plugin_template@2021.04.08-84-g3271e88
+
 additional_repos:
-- branch: master
+- branch: 1.8
   name: pulp_file
-- branch: master
+- branch: 1.4
```
Comment (dralley) [inline template_config.yml:9]: Question: is there anything preventing us from re-ordering these keys in the template so that name comes first?

**Review by dralley — APPROVED** [review]

File: template_config.yml (hunk 2 above)
Comment (daviddavis) [reply template_config.yml:9]: I've tried but it's a losing battle: the next time the plugin-template is run, reverse the ordering again.

Comment (dralley) [reply template_config.yml:9]: Exactly - can we change it in the template?

Comment (daviddavis) [reply template_config.yml:9]: I suppose we could. I'd have to look at the code. Feel free to file an issue.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**